src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -220,10 +220,18 @@ def _initialize_scale_zero_point(
         register_offload_parameter(module, f"{base_name}_g_idx", init_g_idx)
 
 
-def _initialize_attn_scales(module: Module) -> None:
+def _initialize_attn_scales(module: Module, quantization_args: QuantizationArgs) -> None:
     """Initlaize k_scale, v_scale for  self_attn"""
 
-    expected_shape = 1  # per tensor
+    if quantization_args.strategy == QuantizationStrategy.CHANNEL:
+        expected_shape = module.k_proj.out_features
```

Comment:
If channel wise quantization - this should be 2D 
Please refer to the init here; https://github.com/neuralmagic/compressed-tensors/blob/3d49764c02d4d9437e59d35f8c49abb5bc94636c/src/compressed_tensors/quantization/lifecycle/initialize.py#L175